document_test.go

```diff
@@ -198,15 +235,13 @@ func TestDocument_RenderAndReload_ChangeCheck_Stripe(t *testing.T) {
 	tc := compReport.TotalChanges()
 	bc := compReport.TotalBreakingChanges()
 	assert.Equal(t, 0, bc)
-	assert.Equal(t, 820, tc)
+	assert.Equal(t, 819, tc)
 
 	// there should be no other changes than the 519 descriptions.
-	assert.Equal(t, 1, len(filtered))
-
+	assert.Equal(t, 0, len(filtered))
```

Comment:
what was the difference?

Reply:
https://github.com/pb33f/libopenapi/pull/205#discussion_r1426586957